src/Symfony/Component/HttpKernel/DataCollector/EventDataCollector.php

```diff
@@ -15,93 +15,15 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcherInterface;
+use Symfony\Component\Profiler\DataCollector\EventDataCollector as BaseEventDataCollector;
 
 /**
  * EventDataCollector.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ * @deprecated since x.x, to be removed in x.x. Use Symfony\Component\Profiler\DataCollector\EventDataCollector instead.
  */
-class EventDataCollector extends DataCollector implements LateDataCollectorInterface
+class EventDataCollector extends BaseEventDataCollector
```

Comment:
Already fixed this, as I noticed that I can't have the old DataCollectors extend the new ones due to BC issues. 
Because the new ones don't hold any data and have no accessor for them which breaks BC.